impeller/entity/contents/content_context.cc

```diff
@@ -263,7 +263,17 @@ ContentContext::ContentContext(
     desc.size = ISize{1, 1};
     empty_texture_ = GetContext()->GetResourceAllocator()->CreateTexture(desc);
     auto data = Color::BlackTransparent().ToR8G8B8A8();
-    if (!empty_texture_->SetContents(data.data(), 4)) {
+    auto cmd_buffer = GetContext()->CreateCommandBuffer();
```

Comment:
Wanted to change a few places to use the blitpass to make sure it works. We should eventually remove Texture::SetContents.